### PR TITLE
feat: add google and posthog analytics

### LINKED ISSUE #763: [Ember-Migration] Add google and posthog analytics
### Issue Description

We have to add Google and Posthog analytics to get analytics of our web app.

### Expected Behavior

We have to add Google and Posthog analytics to get analytics of our web app.

### Current Behavior

We have to add Google and Posthog analytics to get analytics of our web app.

### Screenshots

[If applicable, add screenshots or images that help illustrate the issue. You can use the following Markdown syntax to embed images:]

### Reproducibility (n/a)

 - [ ] This issue is reproducible
-  [ ] This issue is not reproducible

### Steps to Reproduce
n/a

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [x] Medium
 - [ ] Low

### Additional Information

n/a

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by satyam73**
### What is the change?

Added Google and Posthog analytics to get analytics of our web app.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
n/a

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,25 +1,98 @@
...
-</html>
\ No newline at end of file
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-YRG12HRZKB");
```
Comment (sahsisunny) [inline app/index.html:88]: What is this 'G-YRG12HRZKB'? Is it a key or an ID?

File: app/index.html [hunk 2]
```diff
@@ -1,25 +1,98 @@
...
+            )
+              g(u, o[n]);
+            e._i.push([i, s, a]);
+          }),
+          (e.__SV = 1));
+      })(document, window.posthog || []);
+      posthog.init("phc_SRQcmzRDT5aNlAEkcrGgU2FqxKRaec66c7tdrntqAbs", {
+        api_host: "https://app.posthog.com",
+      });
+    </script>
```
Comment (sahsisunny) [inline app/index.html:73]: Is this script provided by Google?

**Review by sahsisunny — APPROVED** [review]

File: app/index.html [hunk 3]
```diff
@@ -1,25 +1,98 @@
...
+                n = 0;
+              n < o.length;
+              n++
+            )
+              g(u, o[n]);
+            e._i.push([i, s, a]);
+          }),
+          (e.__SV = 1));
+      })(document, window.posthog || []);
+      posthog.init("phc_SRQcmzRDT5aNlAEkcrGgU2FqxKRaec66c7tdrntqAbs", {
```
Comment (prakashchoudhary07) [inline app/index.html:70]: Is this RDS posthog id?

File: app/index.html (hunk 1 above)
Comment (prakashchoudhary07) [reply app/index.html:88]: +1, is this RDS key ?

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: app/index.html (hunk 3 above)
Comment (satyam73) [reply app/index.html:70]: Yes used from vanilla website-www's code

File: app/index.html (hunk 2 above)
Comment (satyam73) [reply app/index.html:73]: Yes

File: app/index.html (hunk 1 above)
Comment (satyam73) [reply app/index.html:88]: Yes this is RDS key, it is known as measurement id

**Comment by satyam73** [comment]
Update - Change keys from RDS official to my account with a TODO when we will release then will replace at that time.

File: app/index.html (hunk 3 above)
Comment (satyam73) [reply app/index.html]: Update - https://github.com/Real-Dev-Squad/website-www/pull/769#issuecomment-1856078585

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**